crates/optimism/payload/src/payload.rs

```diff
@@ -33,12 +34,23 @@ pub struct OptimismPayloadBuilderAttributes {
     pub payload_attributes: EthPayloadBuilderAttributes,
     /// `NoTxPool` option for the generated payload
     pub no_tx_pool: bool,
-    /// Transactions for the generated payload
+    /// Decoded Transactions for the generated payload
     pub transactions: Vec<TransactionSigned>,
+    /// Encoded Transactions for the generated payload
+    pub encoded_transactions: Vec<Bytes>,
```

Comment:
I'd like if we could use a single for for this:

```rust
transactions: Vec<WithEncoded<TransactionSigned>>
```

Reply:
I tried this as well and since I changed this line to use the WithEncoded I had to update other lines of code affected by this.